akka-http2-support/src/main/scala/akka/http/scaladsl/model/http2/Http2StreamIdHeader.scala

```diff
@@ -6,6 +6,7 @@ package akka.http.scaladsl.model.http2
 
 import akka.http.impl.engine.ws.InternalCustomHeader
 
+// TODO should the streamId not be a Long? I may be mis-remembering the spec
```

Comment:
according to the spec:

> Stream Identifier:  A stream identifier (see Section 5.1.1) expressed as an unsigned 31-bit integer

31-bit is really akward, and since it's so small it's actually possible to exhaust it.